html/quote-generator.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25" />
+    <link rel="stylesheet" href="../css/quote-generator.css">
+    <link rel="stylesheet" href="../css/normalize.css">
+    <title>Quote-generator</title>
+</head>
+
+<body>
+    <main class="page">
+        <div class="top-buttons">
+            <div>
+                <a class="top-button" id="all-authors-btn">all authors</a>
```

Comment:
That's interesting case. In general, <a> tags without href is an invalid markup. It means that you need to set href for this attribute. Due to this, focus operation (tab key) to this link doesn't work, because it's invalid

Create 2 html pages:
index.html (qoutes page)
all-authors.html (authors page)

and use `href` attribute to forward user to these pages